charts/elastic/1.3.0/templates/enterprise-search.yaml

```
@@ -0,0 +1,78 @@
+{{- if and .Values.enterprise_search.enabled .Values.kibana.enabled  }}
+apiVersion: enterprisesearch.k8s.elastic.co/v1
+kind: EnterpriseSearch
+metadata:
+  name: {{ include "common.names.shortname" $ }}-enterprise-search
+  namespace: {{ include "common.names.namespace" . }} 
+  labels:
+    {{- include "common.labels.labels" $ | nindent 4 }}
+spec:
+  version: {{ default $.Chart.AppVersion }}
+  image: docker.elastic.co/enterprise-search/enterprise-search:{{  default $.Chart.AppVersion }}
+  count: {{ .Values.enterprise_search.params.count }}
+  elasticsearchRef:
+    name: {{ include "common.names.shortname" $ }}
+  podTemplate:
+    metadata:
+      labels:
+        scrape: entsearch-{{ include "common.names.namespace" $ }}
+    spec:
+      containers:
+      - name: enterprise-search
+        resources:
+          requests:
+            cpu: {{ .Values.enterprise_search.params.cpu }}
+            memory: {{ .Values.enterprise_search.params.memory }}
+        env:
+          - name: JAVA_OPTS
+            value: -Xms{{ .Values.enterprise_search.params.xms }} -Xmx{{ .Values.enterprise_search.params.xmx }}
+  config:
+    # define the exposed URL at which users will reach Enterprise Search
+    ent_search.external_url: https://{{ .Values.enterprise_search.params.ingress.hostname }}
```

```suggestion
    ent_search.external_url: https://{{ .Values.enterprise_search.params.ingress.hostname }}
    enterpriseSearch.ssl.verificationMode: none
```
https://www.elastic.co/guide/en/enterprise-search/current/configure-ssl-tls.html#configure-ssl-tls-in-kibana